### DISCUSSION TITLE
Grafana agent pushing metrics to aws-otel-collector

### DISCUSSION THREAD

**Discussion by parmarhiiren** [grafana-cold-storage/agent#814] [Q&A]
Hi,

I have no prior experience with observability stack. I have few questions in mind and would appreciate the intelligent minds would be able to answer..
1.) is there a way grafana agent can push the metrics (application metrics) to aws-otel-collector ( https://github.com/aws-observability/aws-otel-collector) ? 
2.) is it even possible to push grafana agent metrics ( application ) to the aws-otel-collector?
3.) I can remote_write application metrics to cortex however i want the metrics to be pushed to the collector. if there are any examples can anyone please share it ?

Thanks in advance

**Comment by rfratto** [comment]
Hi, it doesn't look like aws-otel-collector comes with any receiver that implements the Prometheus Remote Write protocol. The Grafana Agent can only send metrics using that protocol, so it'd be required on the receiving end to work.

**Reply by parmarhiiren** [reply]
Thank you for your response.

**Comment by verygoodsoftwarenotvirus** [comment]
Actually, it is possible to use the AWS OTel Collector for `prometheusremotewrite`, as [an engineer noted to me on Twitter](https://twitter.com/Aneurysm9/status/1471332913346224129), you just have to use `awsprometheusremotewrite` instead, and you cannot include basic auth fields in the config part. The basic auth credentials must be baked into the rewrite URL, but it does work.
